e2e_tests/e2e_test.go

```diff
@@ -0,0 +1,794 @@
+package e2e_tests
```

Comment:
I would call these integration tests and not end to end since they don't include the o11y product.

Reply:
Done in #949